zkevm-circuits/src/evm_circuit/step.rs

```diff
@@ -270,11 +271,13 @@ impl ExecutionState {
             Self::BLOCKCTXU160 => vec![OpcodeId::COINBASE],
             Self::BLOCKCTXU256 => {
                 if cfg!(feature = "scroll") {
-                    vec![OpcodeId::DIFFICULTY]
+                    vec![OpcodeId::BASEFEE]
```

Comment:
If our purpose is re-enable BASEFEE, maybe we can just purge the feature option here so the `BLOCKCTXU256` can handle both of these two op codes. And other change should be not required.